lib/companies_web/router.ex

```diff
@@ -25,6 +26,13 @@ defmodule CompaniesWeb.Router do
   scope "/", CompaniesWeb do
     pipe_through [:browser]
 
+    get "/", CompanyController, :recent
```

Comment:
Will we ever hit these routes with the `SetLocale` plug?

Reply:
I use this solution.